src/update.rs

```diff
@@ -83,6 +83,28 @@ pub fn update_bugzilla(
     Ok(profile)
 }
 
+pub fn update_slack(
+    slack_uri: String,
+    slack_username: String,
+    mut profile: Profile,
+    store: &SecretStore,
+) -> Result<Profile, Error> {
+    let now = &Utc::now().to_rfc3339_opts(SecondsFormat::Secs, true);
+    let mut key_val: String = String::from("EA#SLACK#n");
+    if let Some(KeyValue(ref mut values)) = &mut profile.uris.values {
+        for (k, _) in values.into_iter() {
```

Comment:
Clippy is unhappy with this: https://travis-ci.com/mozilla-iam/dino-park-whoami/jobs/243979713#L214